lib/checkPullRequestBranch.js

```diff
@@ -23,28 +22,88 @@ const checkBranch = async (context) => {
     );
+    
     const commentParams = context.repo({
       issue_number: pullRequest.number,
       body:
         'Hi @' + prAuthor +
-        ', PRs made from develop branch or from a branch ' +
+        ', PRs made from develop branch are not allowed. Also branches ' +
         'whose name is prefixed with develop, release or test are not ' +
         'allowed. So this PR is being closed. Please make your changes in ' +
         'another branch and send in the PR. To learn more about contributing ' +
         'to Oppia, take a look at our ' +
         wiki + ' (Rule 1 specifically). Thanks!',
     });
+
+    await context.github.issues.createComment(commentParams);
+    // Close the pull request.
+    const closeIssueParams = context.repo({
+      issue_number: pullRequest.number,
+      state: 'closed',
+    });
+    await context.github.issues.update(closeIssueParams);
+  }
+
+  else if (branchName.startsWith('develop')) {
+    const prAuthor = pullRequest.user.login;
+    // Comment on the pull request.
+    const wiki = 'wiki'.link(
+      'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
+      'instructions-for-making-a-code-change'
+    );
+    
+    const commentParams = context.repo({
+      issue_number: pullRequest.number,
+      body:
+        'Hi @' + prAuthor +
+        ', PRs made from a branch whose name is prefixed with develop ' +
+        'are not allowed. Also PRs made from develop branch or from a ' +
+        'branch whose name is prefixed with release or test are not '+
+        'allowed. So this PR is being closed. Please make your changes in ' +
+        'another branch and send in the PR. To learn more about contributing ' +
+        'to Oppia, take a look at our ' +
+        wiki + ' (Rule 1 specifically). Thanks!',
+    });
+
+    await context.github.issues.createComment(commentParams);
+    // Close the pull request.
+    const closeIssueParams = context.repo({
+      issue_number: pullRequest.number,
+      state: 'closed',
+    });
+    await context.github.issues.update(closeIssueParams);
+  }
+
+  else if (branchName.startsWith('release')) {
+    const prAuthor = pullRequest.user.login;
+    // Comment on the pull request.
+    const wiki = 'wiki'.link(
+      'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
+      'instructions-for-making-a-code-change'
+    );
+    
+    const commentParams = context.repo({
+      issue_number: pullRequest.number,
+      body:
+        'Hi @' + prAuthor +
+        ', PRs made from a branch whose name is prefixed with release ' +
+        'are not allowed. Also PRs made from develop branch or from a ' +
+        'branch whose name is prefixed with develop or test are not '+
+        'allowed. So this PR is being closed. Please make your changes in ' +
+        'another branch and send in the PR. To learn more about contributing ' +
+        'to Oppia, take a look at our ' +
+        wiki + ' (Rule 1 specifically). Thanks!',
+    });
+
```

Comment:
So the point of splitting this into different cases is to have different messages for them. 
For example, if the branch starts with `test`, the message should be `Hi  @author, PRs made from branch whose name is prefixed with test is not allowed. So this PR is being closed...`
And similar for other prefixes.